lib/views/frontend/spree/checkout/payment/_paypal_commerce_platform.html.erb

```diff
@@ -1,6 +1,9 @@
 <script src="<%= payment_method.javascript_sdk_url(order: @order) %>"></script>
 
 <div id="paypal-button-container"></div>
+
+<div data-pp-message data-pp-placement="payment" data-pp-amount="<%= @order.total %>"></div>
```

Comment:
Does this and the following `<div>` tags need to be guarded by an `if options[:display_credit_messaging]` clause?

Reply:
Nope! As long as the credit messaging component isn't loaded in (which is what the `display_credit_messaging` controls) then nothing gets displayed in the div, so it should be safe to leave in place.